okteto.yaml

```diff
@@ -0,0 +1,18 @@
+build:
+  guestbook:
+    dockerfile: Dockerfile
+  dev:
+    dockerfile: Dockerfile
+
+deploy:
+  - cd manifests; kustomize edit set image ${OKTETO_BUILD_GUESTBOOK_IMAGE}
+  - kubectl apply -k manifests
+dev:
+  guestbook:
+    command: ["bash"]
+    forward:
+      - 8080:8080
+      - 9229:9229
+    image: ${OKTETO_BUILD_DEV_IMAGE}
```

Comment:
you don't need this one. that way, it will reuse the same image as in the deployment manifest, which is `OKTETO_BUILD_GUESTBOOK_IMAGE`